packages/cli/internal/pkg/cli/workflow/workflow_run.go

```diff
@@ -22,11 +22,15 @@ func (m *Manager) RunWorkflow(contextName, workflowName, argumentsUrl string) (s
 	m.readInput(argumentsUrl)
 	m.uploadInputsToS3()
 	m.parseInputToArguments()
+	m.readOptionFile(optionFileUrl)
+	m.uploadOptionFileToS3()
+	m.readEngineOptions(engineOptions)
```

Comment:
I'm going to make a separate issue for this because it looks like this is not actually an equivalent to optionFileUrl for Cromwell

Reply:
In that case, let's remove all engineOptions related code.

Reply:
This is implemented as we understand engineOptions to work. The new issue will be to investigate this because from my research, this is not quite right.